CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **Breaking change**: Removed support for matrix formulas (`{=FORMULA}`) notation. Engine now supports formulas returning array of values (instead of only scalars). (#652)
 - **Breaking change**: Removed numeric matrix detection along with matrixDetection and matrixDetectionThreshold config options. (#669)
 - Changed SWITCH function so it takes array as its first argument.
+- Removed GPU acceleration from Known Issues
+- Added TRANSPOSE to the GPU acceleration section 
```

Comment:
IMO we shouldn't add docs fixes to the changelog. In changelog I would expect product changes that may affect the application from user point of view

Reply:
Sure, makes sense! will update